.spelling

```diff
@@ -49,6 +49,7 @@ Slurm
 EX-1C
 CSM-Diags
 Loftsman
+Supercomputing
```

Comment:
Added this exception since "HPE Cray Supercomputing" will be part of all product names/guide titles with the rebrand effort.